MvvmCross.Forms/Presenters/MvxFormsPagePresenter.cs

```diff
@@ -958,5 +962,27 @@ public virtual void ReplacePageRoot(Page existingPage, Page page, MvxPagePresent
                 throw new MvxException(ex, "Cannot replace MainPage root");
             }
         }
+
+        protected virtual Task<bool> FindAndCloseViewFromViewModel(IMvxViewModel mvxViewModel, Page rootPage, MvxPagePresentationAttribute attribute)
+        {
+            var root = TopNavigationPage();
+            Page pageToClose = null;
+
+            if (root?.Navigation?.NavigationStack != null)
```

Comment:
Do we need this null check - can't we propagate the null through to the next line?